### PR TITLE
Bind artifact_prefix during fuzzing for minijail.

### PR DESCRIPTION
**Pull request by oliverchang**
Bind artifact_prefix during fuzzing for minijail.

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR to fix an exception.

File: src/python/bot/fuzzers/libfuzzer.py [hunk 1]
```diff
@@ -575,15 +584,22 @@ def fuzz(self,
...
+    if artifact_prefix:
+      bind_directories.append(artifact_prefix)
+
+    self._bind_corpus_dirs(bind_directories)
     corpus_directories = self._get_chroot_corpus_paths(corpus_directories)
 
+    if artifact_prefix:
+      artifact_prefix = self._get_chroot_directory(artifact_prefix)
+
     # Set artifact prefix to '/' in minijail.
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/libfuzzer.py]: Probably this comment should be removed.

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: Ignore this is fixed in https://github.com/google/clusterfuzz/pull/963#pullrequestreview-286285871

Comment (inferno-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: Ignore; this is fixed in 
https://github.com/google/clusterfuzz/pull/963#pullrequestreview-286285871

**Pull request merged**